apps/01PigLatin.js

```diff
@@ -4,13 +4,67 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // Your Answer:
+  //.toLowerCase makes each letter of a string a lowercase letter
```

Comment:
True, but we are asking for the "why", not the "what".

![why](http://cdn1.theodysseyonline.com/files/2015/09/06/6357714895416725541081155016_not-cool.gif)